docs/content/manual/pre-release/ha/ha-volume-migration.md

```diff
@@ -0,0 +1,31 @@
+---
+title: HA Volume Migration
+---
+
+1. Create a migratable volume:
+    1. Deploy a migratable StorageClass. e.g., https://github.com/longhorn/longhorn/blob/master/examples/rwx/storageclass-migratable.yaml
+    2. Create a PVC with access mode `ReadWriteMany` via this StorageClass.
+2. Attach a volume to a node and wait for volume running. Then write some data into the volume. Here I would recommend directly restoring a volume (set `fromBackup` in the StorageClass) and attach it instead.
+3. Start the migration by request attaching to another node for the volume.
+4. Trigger the following scenarios then confirm or rollback the migration:
+
+   --------------------------
+   | **#** | **Scenario**          | **Migration Confirmation** | **Migration Rollback** |
+   | ----- | --------------------- | -------------------------- | ---------------------- |
+   | 1     | New engine crash      | If the request is successfully sent before the migration engine & replicas being removed, the volume should be auto reattached after migration. <br> Otherwise, the request will be rejected until the volume is reattached and restarts the migration components | Should succeed |
+   | 2     | All new replica crash | If the request is successfully sent before the migration engine & replicas being removed, the volume should keep running (then may start rebuilding) after migration. <br> Otherwise, the request will be rejected until the volume is reattached and restarts the migration components | Should succeed |
+   | 3     | Old engine crash      | If the request is successfully sent before the volume becoming Faulted, the volume should keep running after migration. <br> Otherwise, the request will be rejected until the volume is reattached and restarts the migration components | Should succeed |
+   | 4     | All old replica crash | If the request is successfully sent before the volume becoming Faulted, the volume should keep running (then may start rebuilding) after migration. <br> Otherwise, the request will be rejected until the volume is reattached and restarts the migration components | Should succeed |
+   | 5     | Degraded volume       | Should succeed | Should succeed |
+   | 6     | One replica node down | Should succeed | Should succeed |
+   | 7     | New engine node down  | The request should be rejected | Should succeed |
+   | 8     | Old engine node down  | The request should be rejected | The request should be accepted. But the volume should keep state Unknown after the rollback since it's on a down node. |
+   
```

Comment:
Can there be a case where disk space is not available for the new replicas?

Reply:
NO. All migration replica CRs are directly cloned from the old replicas without regular scheduling progress. And each migration replica will directly use the old replica data directory rather than copying all data.